src/docs/Doxyfile

```diff
@@ -303,7 +303,7 @@ ALIASES                = "arch_page_table{2}=<div class= arch_head><table><tr><t
 # A mapping has the form "name=value". For example adding "class=itcl::class"
 # will allow you to use the command class in the itcl::class meaning.
 
-TCL_SUBST              =
+#TCL_SUBST              =
```

Comment:
It's my habit to comment out things until it's absolutely clear they should be removed, just in case something unexpected comes up. If nothing does, it's easy enough to just delete it all the next time you happen to look at the file a year later. However, in this case I guess there's no reason to keep it around: doxygen said it's no longer supported and it wasn't being used. So I'll delete it.